gmime/envelope.go

```diff
@@ -69,6 +69,29 @@ func (m *Envelope) Headers() textproto.MIMEHeader {
 	return goHeaders
 }
 
+// ReplaceHeader replaces the header with matching key & originalValue with replaceValue
+func (m *Envelope) ReplaceHeader(key, originalValue, replaceValue string) error {
```

Comment:
can you run this in a forloop and check for memory leaks?

Reply:
made a fix. memory usage is constant (around 6.5MB) in forloop now